HeterogeneousCore/AlpakaInterface/test/alpaka/testIndependentKernel.dev.cc

```diff
@@ -121,10 +121,7 @@ TEST_CASE("Test alpaka kernels for the " EDM_STRINGIZE(ALPAKA_ACCELERATOR_NAMESP
   SECTION("Independent work groups") {
     // get the list of devices on the current platform
     auto const& devices = cms::alpakatools::devices<Platform>();
-    if (devices.empty()) {
-      INFO("No devices available on the platform " EDM_STRINGIZE(ALPAKA_ACCELERATOR_NAMESPACE));
-      REQUIRE(not devices.empty());
-    }
+    REQUIRE(!devices.empty());
```

Comment:
Can you revert this change ?
I liked the more explicit explanatory message better.